tunnistamo/middleware.py

```diff
@@ -26,6 +28,18 @@ def add_params_to_url(url, params):
     return url_parts._replace(query=urlencode(new_query_parts)).geturl()
 
 
+class LocaleMiddleware(DjangoLocaleMiddleware):
+    def process_request(self, request):
+        ui_locales = request.GET.get('ui_locales')
+
+        try:
+            language = get_supported_language_variant(ui_locales)
```

Comment:
The other functions from the `translation` module are called like `translation.function_name` but this is not. Maybe this could be called like that too?

Reply:
Indeed that was the case. How silly. Now all functions are used the same way.